### ISSUE TITLE
Cannot load_model

### ISSUE THREAD

**Issue by abhisheksoni27** [keras-team/keras#11023]
Thank you!

- [ ] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps

- [x] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

I am using Google Colab to train a CNN and then save the entire model to a `.h5` file. The code is available here: [CNN-Colab](https://gist.github.com/abhisheksoni27/184c49ca703eb124e1b17eb8dd8af518)

The model gets saved but when I later try to load it back, I get the following error:

```
TypeError: float() argument must be a string or a number, not 'dict'
```

The entire Output log is here: [CNN - Colab - Error](https://gist.github.com/abhisheksoni27/732bec240629d2dd721e80130cb2956b)

**Comment by yanboliang** [comment]
@abhisheksoni27 I think you hit [this](https://github.com/keras-team/keras/issues/10718) bug, it has been fix at ```keras-team/keras```, but you are using ```tf.keras```. Actually I have sent a [fix](https://github.com/tensorflow/tensorflow/pull/21005) to ```tf.keras``` as well, but not merged until now.

**Comment by ymodak** [comment]
Closing this issue since the fixed has been merged. Feel free to reopen the issue if the problem still persists. Thanks!

**Comment by psoulos** [comment]
Is there a way to apply the fix as a patch to Tensorflow 1.12 installed in Anaconda?
